bin/start-prometheus

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
It seems like this script should be either (ideal) rolled into other scripts as part of creating / updating a Harmony environment or otherwise documented as a step in the README.

Reply:
I'm actively making changes to the deployment scripts to fix some Harmony-in-a-box things regarding turbo, so it would be better for me if these don't get rolled into existing scripts right now.

Reply:
👍  I'm gonna leave the scripting for later tickets (once the metrics server and HPA tickets are complete since they'll likely all get deployed together).